README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Who is this book for?
 
-This book is targeted towards experienced programmers that already feel somewhat comfortable with vanilla Rust (you definitely do not need to be an "expert" though, I certainly am not) and would like to dip their toes into its async ecosystem.
+This book is targeted towards experienced programmers that already feel somewhat comfortable with vanilla Rust (you definitely do not need to be an "expert," though; I certainly am not) and would like to dip their toes into its async ecosystem.
```

Comment:
I'll leave more info in comments.

<independent clause>, <independent clause>
is a "comma splice."

Reply:
I am extremely disturbed by that comma being inside the quotes: `"expert,"`.
Is that normal..?

Reply:
Yes. Putting the comma inside the quote in normal. Reading https://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks suggests that it is standard in the US, but optional elsewhere (which I didn't know until now).

Reply:
You know, I actually *hate* that that's correct.  BLECH!  And I refuse to do it when communicating what to type, as in

   At the prompt, type "rm foo."

But elsewhere, I grudgingly follow the rules.  Honestly, if you don't want that one, I don't blame you.  Let me know.  Or actually, I remember checking some box that I *think* allows you to edit my version, so feel free.